src/test/scala/uk/gov/nationalarchives/omega/api/repository/OmegaRepositorySpec.scala

```diff
@@ -43,20 +44,20 @@ class OmegaRepositorySpec extends AsyncUnitTest {
   "Get Legal Status summaries" - {
 
     "must return a Success with an empty list" in {
-      when(mockConnector.execute[LegalStatusSummary](any, any)).thenReturn(Success(List.empty))
+      when(mockConnector.execute[LegalStatusSummary](any, any)).thenReturn(IO.pure(List.empty))
       val result = repository.getLegalStatusEntities
       result.asserting(_.length mustBe 0)
     }
     "must return a Success with a list of one item" in {
       when(mockConnector.execute[LegalStatusSummary](any, any)).thenReturn(
-        Success(List(LegalStatusSummary(new URI(s"${Cat.NS}public-record"), "Public Record")))
+        IO.pure(List(LegalStatusSummary(new URI(s"${Cat.NS}public-record"), "Public Record")))
```

Comment:
According to [the spec](https://typelevel.org/cats-effect/api/3.x/cats/effect/IO$.html#pure[A](value:A):cats.effect.IO[A]), `IO.pure` should only be used where the value has already been computed. In this case I would use `IO.delay` as it is a non-blocking operation and, as `IO.apply` is an alias for `IO.delay`, you can just do: `IO(List(LegalStatusSummary(new URI(s"${Cat.NS}public-record"), "Public Record")))`

Reply:
Good point @rwalpole. I have now addressed that.